trace_context/HTTP_HEADER_FORMAT.md

```diff
@@ -145,16 +145,43 @@ base16(<TraceOptions>) = 00  // not-sampled
 
 # TraceState field
 
+The `tracestate` HTTP header field conveys information about request position in multiple distributed tracing graphs.
+
 ## Header name
 
 `tracestate`
 
 ## Header value
 
-`vendorName1=opaqueValue1,vendorName2=opaqueValue2`
+This section uses the Augmented Backus-Naur Form (ABNF) notation of [RFC5234](https://tools.ietf.org/html/rfc5234), including the DIGIT rule from that document. It also includes the OWS rule from [RFC7230](https://www.rfc-editor.org/info/rfc7230).
+
+```
+dictionary  = dict-member 0*128( OWS "," OWS dict-member )
+dict-member = key "=" value
+```
+
+Identifiers are short (up to 256 characters) textual identifiers.
 
-The value a concatenation of trace graph name-state pairs. Only one entry per
-name is allowed because the entry represents that last position in the trace.
+```
+key = lcalpha 0*255( lcalpha / DIGIT / "_" / "-"/ "*" / "/" )
+lcalpha    = %x61-7A ; a-z
+```
+
+Note that identifiers can only contain lowercase letters.
+
+Valus is opaque string up to 256 characters printable ASCII [RFC0020](https://www.rfc-editor.org/info/rfc20) characters (i.e., the range 0x20 to 0x7E) except comma `,` and `=`. Note that this also excludes tabs, newlines, carriage returns, etc.
```

Comment:
👍 for the layman-readable definitions immediately surrounding the RFC-style definitions. Nicely done. I think we could use one of these layman definitions for `dictionary` and `dict-member` above. It appears to be the only piece of the puzzle missing a layman definition.